src/complete.rs

```diff
@@ -1,82 +1,160 @@
+//! ABNF Core Rules (RFC5234 B.1.)
+
+use std::ops::{RangeFrom, RangeTo};
+
 use nom::{
-    branch::alt,
-    bytes::complete::{tag, take},
-    character::complete::{self as character, anychar, line_ending},
-    combinator::{opt, recognize, verify},
-    error::ParseError,
+    character::complete::satisfy,
+    combinator::{opt, recognize},
+    error::{ErrorKind, ParseError},
     multi::many0_count,
-    sequence::terminated,
-    IResult,
+    sequence::{pair, terminated},
+    AsChar, Err as OutCome, IResult, InputIter, InputLength, Offset, Slice,
```

Comment:
I choice to import `Err` as `OutCome` cause I think `Err` is very confusing

Reply:
This should be `Outcome`.